compose/compiler/compiler-hosted/integration-tests/src/test/java/androidx/compose/compiler/plugins/kotlin/ControlFlowTransformTests.kt

```diff
@@ -2018,9 +2006,8 @@ class ControlFlowTransformTests : AbstractControlFlowTransformTests() {
             @Composable
             fun Example(x: Int, %composer: Composer?, %changed: Int): Int {
               %composer.startReplaceableGroup(<>, "C(Example):Test.kt")
-              val tmp0 = if (x > 0) {
-                %composer.startReplaceableGroup(<>, "")
-                val tmp4_group =
+              val tmp0 = 
```

Comment:
This line has trailing whitespace, which triggers a lint warning in our CI. Could you please remove it? Sorry, I didn't catch this earlier.